hack/Azure/setup/joinMC.sh

```diff
@@ -36,7 +36,7 @@ metadata:
 type: kubernetes.io/service-account-token
 EOF
 
-echo "Creating member cluster..."
+echo "Creating member cluster CR..."
```

Comment:
what is the 'CR'?

Reply:
the "custom resource", just to clairfy that we are not creating the cluster